docs/setup_xcode.md

```diff
@@ -67,165 +37,120 @@ $ defaults delete com.apple.dt.Xcode
-│   │           └── UserInterfaceState.xcuserstate
-│   └── xcuserdata
-│       └── awdeorio.xcuserdatad
-│           └── xcschemes
-│               └── xcschememanagement.plist
-├── p1_library.cpp
-├── p1_library.h
-├── stats.h
-├── stats_public_test.cpp
-└── stats_tests.cpp
-```
+### Add new files
+<div class="primer-spec-callout info" markdown="1">
+Xcode created `main.cpp` by default.  Skip this subsection your first time through the tutorial.  You can come back to it.
+</div>
 
-## Add existing files
-Next, we'll add our existing source code files to the project.
+Open your project folder by selecting `File` > `Open` > navigate to the `p1-stats` folder and open `p1-stats.xcodeproj`.
 
-Start Xcode and open your project.  Here's how to do that all at once, from the command line:
+<div class="primer-spec-callout info" markdown="1">
+**Pro-tip:** Here's a quick way to open Xcode to a specific project folder from the command line.
 ```console
 $ open p1-stats.xcodeproj
 ```
 {: data-variant="no-line-numbers" }
+</div>
 
-Right-click "p1-stats" in the left side bar.  Select "Add Files to 'p1-stats'".
-
-<img src="images/xcode080.png" width="768px" />
-
-Select all the starter files (press Command-A) and click "Add".
-
-<img src="images/xcode100.png" width="768px" />
-
-You will now see your files in the sidebar on the left.
-
-<img src="images/xcode110.png" width="768px" />
-
-
-## Add new files
-
-EECS 280 project 1 requires us to create two new files: `stats.cpp` and `main.cpp`.
-
-Right-click "p1-stats" in the left side bar.  Select "New File".
+Right-click `p1-stats` in the sidebar.  Select "New File".
 
 <img src="images/xcode115.png" width="768px" />
 
 Select "macOS", "C++ File", and click "Next".
 
 <img src="images/xcode116.png" width="512px" />
 
-Name your file.  Uncheck "Also create a header file".  The first file we'll create for project 1 is `stats.cpp`.
+Name your file, we'll use `stats.cpp` for this example.  Uncheck "Also create a header file".  There's nothing wrong with using "create a header file" if your project needs it.
 
 <img src="images/xcode117.png" width="512px" />
 
 Don't select any targets.  Click "Create".
 
 <img src="images/xcode118.png" width="512px" />
 
-Repeat the previous steps to create another new file, `main.cpp`.
-
-Now you'll see the two new files in the side bar.
+You should see your new file in the sidebar.
 
 <img src="images/xcode119.png" width="768px" />
 
-You'll also see the new files at the command line.
+### Add existing files
+If you have starter files, add them to your project directory.  This example is from EECS 280 Project 1.  Your URL or files might be different.
+
+<div class="primer-spec-callout warning" markdown="1">
+**Pitfall:** Make sure you're in the directory containing your source code.
+```console
+$ ls
+main.cpp  p1-stats.xcodeproj
+```
+</div>
+
+We'll use the terminal to download, unpack, and move starter files into the `src` subdirectory.
```

Comment:
```suggestion
We'll use the terminal to download, unpack, and move starter files into the 'p1-stats' directory
```
It is mentioned that it is "optional" to delete the inner project folder. So currently the options are:
1) p1-stats/
2) p1-stats/p1-stats/

src/ is definitely wrong, but it may be weird to phrase this correctly for everyone.

Reply:
Good catch! #52